src/components/Note/Note.jsx

```diff
@@ -1,44 +1,80 @@
 import React from 'react'
 
-function Note({ note }) {
-  return (
-    <form onSubmit="">
-      <label>
-        Name:
-        <input type="text" value={note.name} />
-      </label>
-      <label>
-        Distiller:
-        <input type="text" value={note.distiller} />
-      </label>
-      <label>
-        Age:
-        <input type="text" value={note.age} />
-      </label>
-      <label>
-        Category:
-        <select>
-          <option value={note.category}>{note.category}</option>
-        </select>
-      </label>
-      <label>
-        Color:
-        <input type="text" value={note.color} />
-      </label>
-      <label>
-        Country:
-        <input type="text" value={note.country} />
-      </label>
-      <label>
-        Rating:
-        <input type="number" value={note.rating} />
-      </label>
-      <label>
-        Tasting Notes:
-        <textarea cols="30" rows="10" value={note.tastingNotes} />
-      </label>
-    </form>
-  )
+class Note extends React.Component {
+  constructor(props) {
+    super(props)
+    const { note } = this.props
+    this.state = {
```

Comment:
I'm not sure if this is a bad idea, but one thing we could do here is just say:
```
this.state = this.props.note
```

For new notes, it will start with empty values, and for existing ones, the props will be passed in for the initial state.. it's basically the same as what you have.

Eventually when we make a call to get these notes, we'll have to decide from which component that will be, but right now we're assuming `App.js` already has all the notes and can pass a single one down to here.